### ISSUE TITLE
Crash with SvelteKit and Vite

### ISSUE THREAD

**Issue by macabeus** [BerkinAKKAYA/svelte-image-gallery#12]
When trying to import this package on a SvelteKit project using Vite, it crashes

```
Unknown file extension ".svelte" for /Users/macabeus/ApenasMeu/gambiconf/gambiconf.github.io/node_modules/svelte-image-gallery/Gallery.svelte
TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".svelte" for /Users/macabeus/ApenasMeu/gambiconf/gambiconf.github.io/node_modules/svelte-image-gallery/Gallery.svelte
    at new NodeError (node:internal/errors:372:5)
    at Object.getFileProtocolModuleFormat [as file:] (node:internal/modules/esm/get_format:76:11)
    at defaultGetFormat (node:internal/modules/esm/get_format:118:38)
    at defaultLoad (node:internal/modules/esm/load:21:20)
    at ESMLoader.load (node:internal/modules/esm/loader:407:26)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:326:22)
    at new ModuleJob (node:internal/modules/esm/module_job:66:26)
    at ESMLoader.#createModuleJob (node:internal/modules/esm/loader:345:17)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:304:34)
    at async ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:82:21)
```

**Comment by fedorkanin** [comment]
Writing this to vite.config.js solved the same problem for me:
``ssr: {
noExternal: ['svelte-image-gallery']
}``

**Comment by ty-e-boyd** [comment]
The true fix for this issue is to add "module" and "svelte" to the package.json, both pointing to index.mjs. @BerkinAKKAYA could you make this change? Otherwise, if you give me permission, I can push the change locally and open a pull request.

**Comment by BerkinAKKAYA** [comment]
Hi @ty-e-boyd, thank you for your contribution! Would be awesome if you can open the pull request @ty-e-boyd

**Comment by ty-e-boyd** [comment]
Awesome thanks! Could you please give me permission to do so? Also, appreciate the package a lot. Been using it for about a year now

**Comment by macabeus** [comment]
@ty-e-boyd The default GitHub permissions are enough to open a pull request.
Just make a fork, push the changes to your fork, and then open a PR here
<img width="483" alt="image" src="https://github.com/BerkinAKKAYA/svelte-image-gallery/assets/9501115/06f1a4ff-33bb-4664-b40a-8f19acea15c1">

**Comment by ty-e-boyd** [comment]
Sweet, just did. Didn't realize you could compare across forks, that's pretty dope. 

Fixed in [PR #13 ](https://github.com/BerkinAKKAYA/svelte-image-gallery/pull/13)
